src/altinn-app-frontend/__mocks__/formDataStateMock.ts

```diff
@@ -1,13 +1,13 @@
-import type { IFormDataState } from "../src/features/form/data/formDataReducer";
+import type { IFormDataState } from '../src/features/form/data/formDataReducer';
```

Comment:
Hmm, do we still have some code that did not get auto formatted with prettier in the previous batch format PR?

Reply:
Seems like it.. I don't know why, but after fixing the merge conflict I just ran `eslint --fix` on `src` and got these changes. We'll just call it eventual consistency.. :wink: